functionary/ui/templates/builder/build_detail.html

```diff
@@ -1,58 +1,56 @@
+        <div class="hstack gap-3 py-2">
+            <h1>
+                <i class="fa fa-wrench fa-xl"></i>
+                {{ build.package.render_name }}
+            </h1>
+            <div>
+                <i class="fa fa-circle-info"></i>
+                <span class="fs-5">{{ build.status }}</span>
+            </div>
         </div>
-        <div class="column">
-            <div class="block">
-                <h1 class="title is-1">
-                    <p>
-                        <i class="fa fa-wrench"></i>
-                        <span>{{ build.package.render_name }}</span>
-                        <i class="is-size-5 fa fa-info pl-4"></i>
-                        <span class="is-size-5 has-text-weight-normal">{{ build.status }}</span>
-                    </p>
-                </h1>
+        <div class="vstack gap-4 py-2">
+            <div>
+                <strong class="fs-4"><i class="fa fa-user fa-sm fa-fw pe-1"></i>Creator:</strong>
+                <p class="ps-4 py-2">
+                    {{ build.creator.first_name }} {{ build.creator.last_name }} <span class="text-muted">({{ build.creator.username }})</span>
+                    at {{ build.created_at }}
+                </p>
             </div>
-            <div class="column has-addons">
-                <div class="field">
-                    <label class="label" for="users">
-                        <i class="fa fa-user"></i>&nbsp;Creator:
-                    </label>
-                    <ul id="users">
-                        <li class="block ml-4">
-                            <p>
-                                {{ build.creator.first_name }} {{ build.creator.last_name }} <span class="has-text-grey-light">({{ build.creator.username }})</span>
-                                at {{ build.created_at }}
-                            </p>
-                        </li>
-                    </ul>
+            {% if build.updated_at %}
+                <div>
+                    <strong class="fs-4"><i class="fa fa-clock-rotate-left fa-sm fa-fw pe-1"></i>Last Updated:</strong>
+                    <p class="ps-4 py-2">{{ build.updated_at }}</p>
                 </div>
-                {% if build.updated_at %}
-                    <div class="field">
-                        <label class="label" for="desc">Last Updated:</label>
-                        <span class="ml-4">{{ build.updated_at }}</span>
-                    </div>
-                {% endif %}
-                {% if build_log.log %}
-                    <div class="field">
-                        <label class="label" for="desc">Build Log:</label>
-                        <pre class="ml-4">{{ build_log.log }}</pre>
+            {% endif %}
+            {% if build_log.log %}
+                <div>
+                    <strong class="fs-4"><i class="fa fa-book fa-sm fa-fw pe-1"></i>Build Log:</strong>
+                    <div class="ps-4">
+                        <div class="form-check py-3">
```

Comment:
Can we add `form-switch` here to make it a switch instead of a checkbox?  Seems like a better fit.